misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueConfig.kt

```diff
@@ -58,5 +58,5 @@ class AwsSqsJobQueueConfig(
    * We use the default retry strategy with SQS, which retries 3 times.
    * As a result, your app could potentially spend 3 x this timeout talking to SQS.
    */
-  val sqs_sending_request_timeout_ms: Int = ClientConfiguration.DEFAULT_REQUEST_TIMEOUT
+  val sqs_sending_request_timeout_ms: Int = 5000
```

Comment:
nit: it's not just sending. also deleting too right?

Reply:
maybe just remove `_sending_` and make it clear in the doc it doesn't apply to `receive` which uses long polling, a common practice with receiving